nemo_skills/inference/server/serve_unified.py

```
@@ -0,0 +1,293 @@
+    parser.add_argument("--speaker_reference", default=None, help="Speaker reference audio path")
+    parser.add_argument("--num_frames_per_inference", type=int, default=None, help="Frames per inference")
+    parser.add_argument("--no_decode_audio", action="store_true", help="Disable audio output")
+
+    # Environment setup
+    parser.add_argument("--code_path", default=None, help="Path to add to PYTHONPATH")
+    parser.add_argument("--hack_path", default=None, help="Path to safetensors patch")
+
+    # Debug
+    parser.add_argument("--debug", action="store_true", help="Enable debug mode")
+
+    args, extra_args = parser.parse_known_args()
```

Ruff (0.15.1)

[warning] Unpacked variable `extra_args` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)